### ISSUE TITLE
Several problems when exporting a simple project with the addon

### ISSUE THREAD

**Issue by etherealxx** [Darnoman/Godot-GlobalInput-Addon#4]
Hi, thanks for making this. It's a neat addon.

I have a simple lightweight project here : https://github.com/etherealxx/simple_cps_counter

It's working as expected when i run the project from godot editor. It will increase the counter when i click outside the window.

However, when i exported it into an executable (with almost default settings), when i launch the exported exe, it will show the splash screen for a while before immediately closes.
![image](https://github.com/Darnoman/Godot-GlobalInput-Addon/assets/64251396/3a188e27-6497-44d1-b67a-baac42704eef)
keep in mind this use the default, unmodified godot 4.2.2 mono stable export template

However. there's a weird anomaly. If i set custom export template to godot 4.2.2 stable export template (**non mono**), it can be opened, it displays the UI. However, it cannot detect the clicks, as expected since the addon needs the mono version to run. So does the addon is the one blocking the app to run? I don't know.

Feel free to clone my small project and try exporting it by yourself, will glad to help if you need more testing.

**Comment by Darnoman** [comment]
That is indeed strange because I am able to export it and run the executable normally without any issues

https://github.com/Darnoman/Godot-GlobalInput-Addon/assets/49286636/c6f959d0-d39c-4436-b851-0d3a149999da

can you run it within the console and see if it gives any errors?

**Comment by Darnoman** [comment]
I lied.. I just tested it out with version 4.2.2 stable mono and I believe it has something to do with version 4.2.2 mono export? The one I did above was with 4.3 beta1. Within the 4.2.2 stable mono, when exported it doesn't export the necessary dotnet files with it , it seems. So i'm guessing that's what's causing the issue. See if you are able to export it on the other version of 4.x and when exported a folder should appear alongside your .exe file
